solutions/swb-reference/integration-tests/tests/isolated/aws-accounts/update.test.ts

```diff
@@ -0,0 +1,65 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+import { resourceTypeToKey } from '@aws/workbench-core-base';
+import _ from 'lodash';
+import ClientSession from '../../../support/clientSession';
+import { AccountHelper } from '../../../support/complex/accountHelper';
+import Account from '../../../support/resources/accounts/account';
+import Setup from '../../../support/setup';
+import HttpError from '../../../support/utils/HttpError';
+import RandomTextGenerator from '../../../support/utils/randomTextGenerator';
+import { checkHttpError } from '../../../support/utils/utilities';
+
+describe('awsAccounts update negative tests', () => {
+  const setup: Setup = new Setup();
+  let adminSession: ClientSession;
+
+  beforeEach(() => {
+    expect.hasAssertions();
+  });
+
+  beforeAll(async () => {
+    adminSession = await setup.getDefaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('when updating an account', () => {
+    let account: Account;
+
+    beforeEach(() => {
+      const accountId = `${resourceTypeToKey.account.toLowerCase()}-00000000-0000-0000-0000-000000000000`;
+      account = adminSession.resources.accounts.account(accountId);
+    });
+
+    describe('and the update params are invalid', () => {
+      test('it throws a validation error', async () => {
+        try {
+          const badValue = 1 as unknown as string;
```

Comment:
Nitpick: rename `badValue` to `invalidValue`. Btw, do we actually need to typecast `1` as string? I didn't see any defined types for an account update request. Without a defined type for account update request, the code would inherit `resource` type definition.
https://github.com/aws-solutions/solution-spark-on-aws/blob/07a4a68c2cd2a6f8457d07fac71447b581454e2b/solutions/swb-reference/integration-tests/support/resources/base/resource.ts#L41

Reply:
the type of body is `{ [key: string]: string }`, so we have to override the type system to allow it to pass the compiler. The hang up here is we're using type validated code to test what should be JSON values. We can solve this by starting to use `axios` directly instead of an SDK, but i'm not sure what else that would entail at the moment.